client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -194,7 +174,10 @@ public Object invoke(Object proxy, Method method, Object[] arguments) {
             throw new IllegalArgumentException("Must be used with observable returned from api call");
         }
 
-        return source.map(data -> new Response<>(((ObservableWithResponse<T>)source).getResponse(), data));
+        return source.map(data -> new Response<>(((ObservableWithResponse<T>)source).getResponse(), data))
+            .switchIfEmpty(Observable.defer(() ->
```

Comment:
You can use `fromCallable` to avoid the `just` on the next line, saving one Observable :-)